views/quiz-result.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Result</title>
+    <link rel="stylesheet" href="../public/css/quiz-result.css">
+</head>
+<body>
+    <nav class="navbar navbar-fixed flex flex-wrap flex-between  navbar-bg nav-zindex">
+        <div class="flex flex-space-evenly flex-align-item-center ">
+            <ul class=" text-deocration-none flex flex-wrap flex-align-item-center gap">
+                <li class="pb-1 nav-text navbar-white-text flex flex-align-item-center gap-1">
+                    <span class="material-icons snackbar-icons  navbar-icons">menu</span>
+                </li>
+                <a href="../views/home.html" class="text-deocration-none list-style-none">
+                    <li class="pb-1  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="font-bold">Quiz-Grad</span>
+                    </li>
+                </a>
+            </ul>
+        </div>
+    
+        <div class="flex flex-space-evenly flex-align-item-center ">
+            <ul class=" text-deocration-none flex flex-wrap gap-col-2">
+               
+                <a href="#" class="text-deocration-none" target="_blank" rel="noreferrer">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons">help_outline</span>
+                        <span class="">help</span>
```

Comment:
remove the unused span class

Reply:
> remove the unused span class

@Shivani-1524 actually I am using google icons so to use them basically have to follow these structures.